src/Application/index.js

```diff
@@ -183,6 +185,14 @@ export default function(App, appData, platformSettings) {
       return initLanguage(file, language)
     }
 
+    loadColors(config) {
+      let colors = Utils.asset('colors.json')
```

Comment:
Nit-picky: but can we call the variable `file` (instead of `colors`), so it's in line with the loadLanguage function?